src/lib/viewers/BaseViewer.js

```diff
@@ -3,6 +3,8 @@ import EventEmitter from 'events';
 import debounce from 'lodash.debounce';
 import fullscreen from '../Fullscreen';
 import RepStatus from '../RepStatus';
+// TODO (@minhnguyen): Import controller from annotations npm module once the library is separate
+import DrawingController from '../annotations/drawing/DrawingController';
```

Comment:
I really want to avoid calling annotation specific code in BaseViewer. Is it possible to move this into Annotator or BoxAnnotations?

Reply:
Good point, I'll move it into annotator as discussed.